build/configs/rtl8721csm/nettest/defconfig

```diff
@@ -0,0 +1,1408 @@
+#
+# CONFIG_MTD_M25P is not set
+# CONFIG_RAMMTD is not set
+CONFIG_MTD_SMART=y
+
+#
+# SMART Device options
+#
+CONFIG_MTD_SMART_SECTOR_SIZE=1024
+CONFIG_MTD_SMART_WEAR_LEVEL=y
+# CONFIG_MTD_SMART_ENABLE_CRC is not set
+# CONFIG_MTD_SMART_JOURNALING is not set
+# CONFIG_MTD_SMART_SECTOR_ERASE_DEBUG is not set
+# CONFIG_MTD_SMART_ALLOC_DEBUG is not set
+# CONFIG_MTD_W25 is not set
+# CONFIG_MTD_JEDEC is not set
+
+#
+# System Logging
+#
+# CONFIG_SYSLOG is not set
+# CONFIG_SYSLOG_TIMESTAMP is not set
+
+#
+# Database
+#
+# CONFIG_ARASTORAGE is not set
+
+#
+# AraUI Framework
+#
+# CONFIG_UI is not set
+
+#
+# Memory Management
+#
+CONFIG_MM_KERNEL_HEAP=y
+# CONFIG_REALLOC_DISABLE_NEIGHBOR_EXTENSION is not set
+# CONFIG_MM_SMALL is not set
+CONFIG_KMM_REGIONS=3
+CONFIG_KMM_NHEAPS=3
+# CONFIG_KMM_FORCE_ALLOC_AT is not set
+# CONFIG_GRAN is not set
+
+#
+# Work Queue Support
+#
+CONFIG_SCHED_WORKQUEUE=y
+
+#
+# Kernel Work Queue
+#
+CONFIG_SCHED_HPWORK=y
+CONFIG_SCHED_HPWORKPRIORITY=224
+CONFIG_SCHED_HPWORKSTACKSIZE=2048
+CONFIG_SCHED_LPWORK=y
+CONFIG_SCHED_LPNTHREADS=1
+CONFIG_SCHED_LPWORKPRIORITY=176
+CONFIG_SCHED_LPWORKSTACKSIZE=2048
+
+#
+# User Work Queue
+#
+CONFIG_SCHED_USRWORK=y
+CONFIG_SCHED_USRWORKPRIORITY=100
+CONFIG_SCHED_USRWORKSTACKSIZE=2048
+
+#
+# Power Management
+#
+# CONFIG_PM is not set
+
+#
+# Debug Options
+#
+# CONFIG_DEBUG is not set
+
+#
+# Stack Debug Options
+#
+CONFIG_ARCH_HAVE_STACKCHECK=y
+# CONFIG_STACK_COLORATION is not set
+
+#
+# Build Debug Options
+#
+# CONFIG_DEBUG_SYMBOLS is not set
+# CONFIG_FRAME_POINTER is not set
+
+#
+# Logger Module
+#
+# CONFIG_LOGM is not set
+
+#
+# System Call
+#
+CONFIG_LIB_SYSCALL=y
+CONFIG_SYS_RESERVED=11
+CONFIG_SYS_NNEST=2
+
+#
+# Built-in Libraries
+#
+
+#
+# Standard C Library Options
+#
+CONFIG_STDIO_BUFFER_SIZE=64
+CONFIG_STDIO_LINEBUFFER=y
+CONFIG_NUNGET_CHARS=2
+CONFIG_LIB_HOMEDIR="/"
+CONFIG_LIBM=y
+# CONFIG_NOPRINTF_FIELDWIDTH is not set
+CONFIG_LIBC_FLOATINGPOINT=y
+CONFIG_LIBC_LONG_LONG=y
+CONFIG_LIBC_FLOATPRECISION=6
+CONFIG_LIBC_SCANSET=y
+# CONFIG_NOPRINTF_LONGLONG_TO_ASCII is not set
+# CONFIG_LIBC_IOCTL_VARIADIC is not set
+# CONFIG_LIBC_WCHAR is not set
+# CONFIG_LIBC_LOCALE is not set
+CONFIG_LIB_RAND_ORDER=1
+# CONFIG_EOL_IS_CR is not set
+# CONFIG_EOL_IS_LF is not set
+# CONFIG_EOL_IS_BOTH_CRLF is not set
+CONFIG_EOL_IS_EITHER_CRLF=y
+CONFIG_LIBC_STRERROR=y
+# CONFIG_LIBC_STRERROR_SHORT is not set
+# CONFIG_LIBC_PERROR_STDOUT is not set
+CONFIG_LIBC_TMPDIR="/tmp"
+CONFIG_LIBC_MAX_TMPFILE=32
+CONFIG_ARCH_LOWPUTC=y
+# CONFIG_LIBC_LOCALTIME is not set
+# CONFIG_TIME_EXTENDED is not set
+CONFIG_LIB_SENDFILE_BUFSIZE=512
+CONFIG_LIBC_ARCH_ELF=y
+CONFIG_ARCH_OPTIMIZED_FUNCTIONS=y
+# CONFIG_ARCH_MEMCPY is not set
+CONFIG_MEMCPY_VIK=y
+CONFIG_MEMCPY_PRE_INC_PTRS=y
+CONFIG_MEMCPY_INDEXED_COPY=y
+# CONFIG_MEMCPY_64BIT is not set
+# CONFIG_ARCH_MEMCMP is not set
+# CONFIG_ARCH_MEMMOVE is not set
+# CONFIG_ARCH_MEMSET is not set
+# CONFIG_MEMSET_OPTSPEED is not set
```

Comment:
For enhancing the memset speed, I enabled the `MEMSET_OPTSPEED` by default.
Please enable this. You can refer to this PR.
https://github.com/Samsung/TizenRT/pull/4719